contrib/guix/patches/binutils-mingw-w64-disable-flags.patch

```diff
@@ -0,0 +1,171 @@
+Description: Add disable opposites to the security-related flags
```

Comment:
Checked that this matches https://salsa.debian.org/mingw-w64-team/binutils-mingw-w64/-/blob/master/debian/patches/disable-flags.patch bar whitespace changes.